apiserver/facades/client/modelconfig/register.go

```diff
@@ -36,17 +36,18 @@ func makeFacade(ctx facade.ModelContext) (*ModelConfigAPI, error) {
 	auth := ctx.Auth()
 
 	domainServices := ctx.DomainServices()
-	modelSecretBackend := domainServices.ModelSecretBackend()
-
-	configService := domainServices.Config()
-	modelService := domainServices.ModelInfo()
 
 	return NewModelConfigAPI(
 		ctx.ModelUUID(),
 		ctx.ControllerUUID(),
 		ctx.State(),
-		modelSecretBackend, configService, modelService, auth,
+		domainServices.ModelSecretBackend(),
+		domainServices.Config(),
+		domainServices.Agent(),
+		domainServices.ModelInfo(),
+		auth,
```

Comment:
Existing code, but can we move auth to the top so we can group the domain services together?

Reply:
Done & Done